README.md

```diff
@@ -17,7 +17,7 @@
 #
 -->
 
-# Apache APISIX
+# The Apache APISIX API Gateway
```

Comment:
Just keep it straightforward; no need to start with `The`.

```suggestion
# Apache APISIX API Gateway
```

Reply:
fixed